tests/robot-tests/tests/admin_and_public/bau/data_catalogue.robot

```diff
@@ -298,3 +299,21 @@ Validate zip contains correct files
     sleep    8    # wait for file to download
     ${list}=    create list    data/dates.csv    data-guidance/data-guidance.txt
     zip should contain directories and files    ui-tests-data-catalogue-%{RUN_IDENTIFIER}_2021-22-q1.zip    ${list}
+
+Validate data catalogue page redirect from slug based urls
+    environment variable should be set    PUBLIC_URL
+    user navigates to public frontend    %{PUBLIC_URL}/data-catalogue/${PUPIL_ABSENCE_PUBLICATION_SLUG}/2016-17?newDesign=true
+    user waits until h1 is visible    Data catalogue
```

Comment:
Happy either way, but I might just be tempted to add an explicit url check using something like 

`user checks url equals    %{PUBLIC_URL}/data-catalogue?publicationId=${PUPIL_ABSENCE_PUBLICATION_ID}&releaseId=2016-17&newDesign=true`

That said, this could fail if the url parameters occur in a different order so maybe we don't actually gain anything from it. Approving either way but wanted to throw the idea out

Reply:
That's a good idea but I'm not sure it's possible as we'd need to know the GUIDs for the publication and release, which, I'm assuming, are generated when the seed data is created so will be different in different environments.